static/app/views/issueDetails/streamline/foldSection.tsx

```diff
@@ -160,23 +160,25 @@ const Content = styled('div')`
   padding: ${space(0.5)} ${space(0.75)};
 `;
 
-const Summary = styled('summary')<{preventCollapse: boolean}>`
+const Summary = styled('div')<{preventCollapse: boolean}>`
```

Comment:
nit: Since it's not a summary anymore would you mine renaming this to something more accurate? I only named it that because it was a component version of `summary`